jwt/src/main/java/me/uport/sdk/jwt/JWTTools.kt

```diff
@@ -209,16 +221,20 @@ class JWTTools(
 
     /**
      * Verifies a jwt [token]
-     * @params jwt token
-     * @throws InvalidJWTException when the current time is not within the time range of payload iat and exp
-     *          when no public key matches are found in the DID document
+     * @param token the jwt token to be verified
+     * @param auth if this param is `true` the public key list that this token is checked against is filtered to the
+     *          entries in the `authentication` entries in the DID document of the issuer of the token.
+     * @param audience the audience that should be able to verify this token. This is usually a DID but can also be a
+     *          callback URL for situations where the token represents a response or a bearer token.
+     * @throws InvalidJWTException when the current time is not within the time range of payload `iat` and `exp`
+     *          , when no public key matches are found in the DID document
+     *          , when the `audience` does not match the intended audience (`aud` field)
      * @return a [JwtPayload] if the verification is successful and `null` if it fails
      */
     suspend fun verify(
         token: String,
         auth: Boolean = false,
-        aud: String? = null,
-        callbackUrl: String? = null
+        audience: String? = null
```

Comment:
`aud` and `callback` were checked using the exact same logic.
There is no point in specifying 2 distinct params.